dir.props

```diff
@@ -71,24 +70,50 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <VersionSuffix Condition="'$(PreReleaseLabel)' != ''">$(PreReleaseLabel)-</VersionSuffix>
     <VersionSuffix>$(VersionSuffix)$(BuildNumberMajor)-$(BuildNumberMinor)</VersionSuffix>
-
-    <ProductVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)-$(VersionSuffix)</ProductVersion>
+    <ProductVersionSuffix Condition="'$(StabilizePackageVersion)' !='true'">-$(VersionSuffix)</ProductVersionSuffix>
+    <ProductVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)$(ProductVersionSuffix)</ProductVersion>
     <ProductionVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)</ProductionVersion>
 
-    <SharedFrameworkNugetVersion Condition="'$(StabilizePackageVersion)' == 'true'">$(ProductionVersion)</SharedFrameworkNugetVersion>
-    <SharedFrameworkNugetVersion Condition="'$(StabilizePackageVersion)' == 'false'">$(ProductVersion)</SharedFrameworkNugetVersion>
+    <SharedFrameworkNugetVersion>$(ProductVersion)</SharedFrameworkNugetVersion>
 
     <NuGetVersion>$(SharedFrameworkNugetVersion)</NuGetVersion>
-    <HostVersion>$(SharedFrameworkNugetVersion)</HostVersion>
-    <AppHostVersion>$(SharedFrameworkNugetVersion)</AppHostVersion>
-    <NetCoreAppVersion>$(SharedFrameworkNugetVersion)</NetCoreAppVersion>
-    <HostResolverVersion>$(SharedFrameworkNugetVersion)</HostResolverVersion>
-    <HostPolicyVersion>$(SharedFrameworkNugetVersion)</HostPolicyVersion>
+
+    <!--
+      By default, we are always building the nuget packages for HostPolicy, HostFXR and
+      Dotnet/AppHost. Thus, the "UseShipped*" properties (below) are always set to false.
+      
+      However, there are scenarios when some of these components will not change (e.g. during
+      servicing, we may only change HostPolicy but not HostFXR and Dotnet/AppHost). In such cases,
+      set the appropriate "UseShipped*" property below to true so that we will use the last shipped
+      version of the package. 
+    -->
+
+    <!-- The host/apphost package versions are only updated whenever there is a change in the components -->
+    <UseShippedHostPackage>false</UseShippedHostPackage>
```

Comment:
Why do we even need these extra properties? Why wouldn't we just have the version and then in servicing change it from SharedFrameworkNugetVersion to whatever hard-coded version we need? It seems kind of redundant to need to set both this bool and the version.